app/views/guides/index.html.erb

```diff
@@ -22,6 +23,14 @@
         <td class='last-edited-by'>
           <%= latest_editor_name(guide) %>
         </td>
+        <td>
+          <% if guide.latest_edition.review_request.present? && guide.latest_edition.review_request.approvals.any? %>
```

Comment:
Would it make sense to encapsulate this logic as `Guide#current_draft_approvals` or something similar?
Looks a bit heavy, especially for a view.